integrations/launchdarkly/client.py

```diff
@@ -0,0 +1,218 @@
+from port_ocean.utils import http_async_client
+import httpx
+from typing import Any, AsyncGenerator, Optional
+from loguru import logger
+from enum import StrEnum
+import asyncio
+
+from port_ocean.context.event import event
+
+PAGE_SIZE = 100
+
+
+class ObjectKind(StrEnum):
+    PROJECT = "project"
+    AUDITLOG = "auditlog"
+    FEATURE_FLAG = "flag"
+    ENVIRONMENT = "environment"
+
+
+class LaunchDarklyClient:
+    def __init__(
+        self, api_token: str, launchdarkly_url: str = "https://app.launchdarkly.com"

```

Comment:
you already specify the default url in the spec.yaml so just use the passed configuration and no need to set a default here as well.